### PR TITLE
Update hot-reload.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
See [this comment](https://developercommunity.visualstudio.com/t/browser-link-cannot-be-disabled/1582653#T-N1608441)

[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/test/hot-reload?view=aspnetcore-6.0&branch=pr-en-us-26064#disable-hot-reload)

### CODE REVIEW

File: aspnetcore/test/hot-reload.md [hunk 1]
```diff
@@ -57,6 +57,14 @@ To disable support for Hot Reload, pass the `--no-hot-reload` option to the `dot
 dotnet watch --no-hot-reload
 ```
 
+## Disable Hot Reload
+
+The following setting in `Properties/launchSettings.json` disable Hot Reload
```
Comment (Rick-Anderson) [inline aspnetcore/test/hot-reload.md:62]:
```suggestion
The following setting in `Properties/launchSettings.json` disables Hot Reload:
```

File: aspnetcore/test/hot-reload.md [hunk 2]
```diff
@@ -57,6 +57,14 @@ To disable support for Hot Reload, pass the `--no-hot-reload` option to the `dot
 dotnet watch --no-hot-reload
 ```
 
+## Disable Hot Reload
+
+The following setting in `Properties/launchSettings.json` disables Hot Reload:
+
+```JSON
```
Comment (guardrex) [inline aspnetcore/test/hot-reload.md:64]:
```suggestion
```json
```

File: aspnetcore/test/hot-reload.md [hunk 3]
```diff
@@ -57,6 +57,14 @@ To disable support for Hot Reload, pass the `--no-hot-reload` option to the `dot
...
+ "hotReloadEnabled" : false
+ ```
```
[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply aspnetcore/test/hot-reload.md:66]: What's the suggestion?

[the remark this answers is not in the document]
Comment (Rick-Anderson) [reply aspnetcore/test/hot-reload.md:66]: You can always edit the fle

[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/test/hot-reload.md:66]: I don't like editing another doc author's PR.

[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/test/hot-reload.md:66]: The suggestion is that there's a single blank space at the start of the line.

**Comment by Rick-Anderson** [comment]
> I don't like editing another doc author's PR.

Ditto, I usually create another PR that merges into the branch.

**Pull request merged**